platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -164,6 +164,10 @@ String OS_LinuxBSD::get_processor_name() const {
 	ERR_FAIL_V_MSG("", String("Couldn't get the CPU model name from `/proc/cpuinfo`. Returning an empty string."));
 }
 
+bool OS_LinuxBSD::is_sandboxed() const {
+	return (getenv("container") || getenv("APPIMAGE") || getenv("SNAP"));
+}
```

Comment:
Would it make sense to unify these checks with https://github.com/godotengine/godot/blob/eb4301b941fa211de204e37bd4d701f7e490a945/platform/linuxbsd/joypad_linux.cpp#L85-L105 so the OS method can be used there too?

Reply:
`container` env. variable seems too ambiguous, and APPIMAGE is not sandboxed, so moved the checks from the `joypad_linux` to the `OS` as is (other apps seems to use the same checks).